Vagrantfile

```diff
@@ -25,6 +25,14 @@ if File.exist?(env_specific_config_path)
   end
 end
 
+# Check that an allowed networking plugin is provided
+allowed_cni_plugins=["weavenet","calico","flannel","no-cni-plugin"]
+if not allowed_cni_plugins.include? settings["ansible"]["group_vars"]["all"]["kubernetes_network_plugin"]
+  puts "kubernetes_network_plugin is not valid in defaults.yaml or env.yaml, allowed values are:"
```

Comment:
We should use the vagrant logger, not puts.

Example:
```ruby
# Define this near the top, we might need it later
@ui = Vagrant::UI::Colored.new
@ui.error 'kubernetes_network_plugin is not valid in defaults.yaml or env.yaml, allowed values are:'
```

Reply:
done, thanks